### PR TITLE
Feature/en 3170 shloka

### PR DESCRIPTION
**Pull request by shloka-bhalgat-unskript**
## Description
Added 2 Runbooks-

1. AWS Access Key Rotation
2. Renew SSL Certificate

### AWS Access Key Rotation Runbook
This runbook has 4 legos-

- Create Access Key
- List Access Keys
- Update Access Keys
- Delete Access Key

Each of which have been placed with a .json, .py, image and readme files in their respective directories.

#### Aim
The aim of this runbook is to allow a user to create a new access key after a certain period of time as per AWS Cloud security guidelines.

### Renew SSL Certificate
This runbook has 2 legos-

- AWS Check SSL Certificate expiry (new)
-  Post Slack message (existing)

The new lego has been placed with a .json, .py, image and readme files in it's respective directory.

#### Aim

The purpose of this runbook is to allow a user to know when/if their SSL certificate is approaching an expiration date and send a reminder to renew it.

### Checklist:
- [ ] Created individual directories for each new lego under the respective connector with a .json, .py and readme file
- [ ] Added the runbook to the main folder of the respective connector 
- [ ] Removed credentials from the .ipynb file

### CODE REVIEW

File: AWS/legos/aws_check_ssl_certificate_expiry/aws_check_ssl_certificate_expiry.py [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    iamClient = handle.client('acm', region_name=region)
+    result = iamClient.describe_certificate(CertificateArn=aws_certificate_arn)
+    for key,value in result['Certificate'].items():
+        if key == "NotAfter":
+            expiry_date = value
+            right_now = datetime.datetime.now(dateutil.tz.tzlocal())
+            diff = expiry_date-right_now
+            days_remaining = diff.days
+            if days_remaining < 30:
+                return days_remaining
```
Comment (jayasimha-raghavan-unskript) [inline AWS/legos/aws_check_ssl_certificate_expiry/aws_check_ssl_certificate_expiry.py:56]: What happens if the days_remaining is more than 30 days, shouldn't we return the number of days.

File: AWS/legos/aws_list_access_keys/aws_list_access_keys.py [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+                :param aws_username: Username of the IAM user to be looked up
+
+                :rtype: Result Dictionary of result
+    """
+    iamClient = handle.client('iam')
+    paginator = iamClient.get_paginator('list_access_keys')
+    for response in paginator.paginate(UserName= aws_username):
+        for key,value in response.items():
+            if key == "AccessKeyMetadata":
+                return value
```
Comment (jayasimha-raghavan-unskript) [inline AWS/legos/aws_list_access_keys/aws_list_access_keys.py:42]: There should be a default return value. If Key is not access key metadata then the Lego will not return anything. This is not a good way of writing Legos.

**Review by jayasimha-raghavan-unskript — CHANGES_REQUESTED** [review]
Please take care of Comments

File: AWS/legos/aws_list_access_keys/aws_list_access_keys.py (hunk 2 above)
Comment (shloka-bhalgat-unskript) [reply AWS/legos/aws_list_access_keys/aws_list_access_keys.py:42]: Screenshot 2022-09-15 at 4 28 10 PM

I was returning only the list of access keys without the response metadata. Will update it to return the entire thing

File: AWS/legos/aws_check_ssl_certificate_expiry/aws_check_ssl_certificate_expiry.py (hunk 1 above)
Comment (shloka-bhalgat-unskript) [reply AWS/legos/aws_check_ssl_certificate_expiry/aws_check_ssl_certificate_expiry.py:56]: Added cases for days > 0, 30 > days < 0, and days < 0.

Days <30:



Days <0 :
Screenshot 2022-09-17 at 7 30 14 PM

Days >0 (Default):
Screenshot 2022-09-17 at 7 30 52 PM

**Review by jayasimha-raghavan-unskript — APPROVED** [review]

**Pull request merged**